blueprints/screening_app/entities/unit/mapping.json

```diff
@@ -0,0 +1,55 @@
+{
+   "entities":[
+      {
+         "id":"54471",
+         "cname":"Kilogramos",
+         "whitelist":[
+            "kilos",
+			   "kilo",
```

Comment:
I think the indentation is this file got messed up!

Reply:
Fixed.